source/index/contracts/Index.sol

```diff
@@ -147,33 +147,47 @@ contract Index is Ownable {
     * @notice Get a Range of Locators
     * @dev start value of 0x0 starts at the head
     * @param start address Identifier to start with
-    * @param count uint256 Number of locators to return
-    * @return bytes32[] result List of locators
+    * @param limit uint256 Number of locators to return
+    * @return bytes32[] List of locators
+    * @return uint256[] List of scores corresponding to locators
+    * @return address The next identifier to provide for pagination
```

Comment:
next cursor? not sure if that's the right usage of the term but replace with something that's not identifier.